homeassistant/components/media_player/mediaroom.py

```diff
@@ -9,134 +9,170 @@
 import voluptuous as vol
 
 from homeassistant.components.media_player import (
-    MEDIA_TYPE_CHANNEL, SUPPORT_PAUSE, SUPPORT_PLAY_MEDIA,
-    SUPPORT_TURN_OFF, SUPPORT_TURN_ON, SUPPORT_STOP, PLATFORM_SCHEMA,
-    SUPPORT_NEXT_TRACK, SUPPORT_PREVIOUS_TRACK, SUPPORT_PLAY,
-    SUPPORT_VOLUME_STEP, SUPPORT_VOLUME_MUTE,
-    MediaPlayerDevice)
+    MEDIA_TYPE_CHANNEL, SUPPORT_PAUSE, SUPPORT_PLAY_MEDIA, SUPPORT_TURN_OFF,
+    SUPPORT_TURN_ON, SUPPORT_STOP, PLATFORM_SCHEMA, SUPPORT_NEXT_TRACK,
+    SUPPORT_PREVIOUS_TRACK, SUPPORT_PLAY, SUPPORT_VOLUME_STEP,
+    SUPPORT_VOLUME_MUTE, MediaPlayerDevice,
+)
+from homeassistant.helpers.dispatcher import (
+    dispatcher_send, async_dispatcher_connect
+)
 from homeassistant.const import (
-    CONF_HOST, CONF_NAME, CONF_OPTIMISTIC, CONF_TIMEOUT,
-    STATE_PAUSED, STATE_PLAYING, STATE_STANDBY,
-    STATE_ON)
+    CONF_HOST, CONF_NAME, CONF_OPTIMISTIC, STATE_OFF,
+    CONF_TIMEOUT, STATE_PAUSED, STATE_PLAYING, STATE_STANDBY,
+    STATE_UNAVAILABLE
+)
 import homeassistant.helpers.config_validation as cv
-REQUIREMENTS = ['pymediaroom==0.5']
+
+REQUIREMENTS = ['pymediaroom==0.6']
 
 _LOGGER = logging.getLogger(__name__)
 
-NOTIFICATION_TITLE = 'Mediaroom Media Player Setup'
-NOTIFICATION_ID = 'mediaroom_notification'
+PLATFORM = "mediaroom"
```

Comment:
What is this? Please remove it as it confuses people in thinking it actually does something.